packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooserBody/__snapshots__/ColumnChooserBody.component.test.js.snap

```diff
@@ -10,17 +10,17 @@ exports[`ColumnChooserBody should render the columns rows and the column select
     <div class="tc-column-chooser-body theme-tc-column-chooser-body">
       <div class="tc-column-chooser-row theme-tc-column-chooser-row tc-column-chooser-row-select-all theme-tc-column-chooser-row-select-all theme-tc-column-chooser-row-select-all theme-tc-column-chooser-row-select-all">
         <div class="checkbox tc-toggle theme-tc-toggle checkbox">
-          <label for="body-context-id-body-checkbox-Columns"
+          <label for="body-context-id-body-checkbox-Select-All"
```

Comment:
Hmm you should check with some QAs in apps that use this component (TDC at least) to see if they use this id in automated tests.
Or just revert the id

Reply:
Hi Jimmy,
Thanks for review!
The id change is inevitable, because it is generated with label before:
```
                        <Checkbox
				checked={checked}
				data-feature={dataFeature}
				aria-describedby={describedby}
				id={`${id}-checkbox-${label}`}
				label={label}
				onChange={onChangeCheckbox}
			/>
```
Even if I removed label from string template, it still change the id.
But I will warn people for this change in tui channel.